extensions/objects/templates/image-editor.component.ts

```diff
@@ -815,10 +815,11 @@ export class ImageEditorComponent implements OnInit, OnChanges {
   }
 
   increaseResizePercent(amount: number): void {
-    // Do not allow to increase size above 100% (only downsize allowed).
+    // Do not allow to increase size above 200% (only downsize allowed).
```

Comment:
This comment doesn't make sense now. "200%" but then "only downsize allowed", could you also explain why we allow more than 100%?

Reply:
I've clarified it now, PTAL!